### ISSUE TITLE
Make composer/packagist compatibile

### ISSUE THREAD

**Issue by rob-mccann** [canton7/fuelphp-casset#31]
This would make it usable in fuel 2.0 :)

**Comment by canton7** [comment]
Sorry for the delay, I was on holiday.

Cheers for the heads-up. I haven't really played with composer, but am I right in thinking that a single `composer.json`, as in d7d453776, is enough?

I'll add it to packagist when that commit gets merged in.

Thanks

**Comment by rob-mccann** [comment]
I'm no expert on this, so take it with a pinch of salt:

I think for fuel 2.0's autoloader, the folder structure needs to follow PSR-0. So folder names and files need to start with caps.

Also, it'd be worth submitting the repo to http://packagist.org so that it can be used with a default composer installation.

Hope that somewhat helps!

**Comment by canton7** [comment]
Yeah, having dug into it a bit more (there's no documentation for fuelphp 2.0 yet), it seems PSR-0 is one of the requirements, along with a set of (as-yet-undefined) fuel package standards.

Given that there are no third-party fuelphp 2.0 packages yet (as far as I can find), and no documentation, I'm tempted to leave this until fuelphp 2.0 is closer to release, and there's more information available. I also don't want to commence on significant re-working of Casset (which would be necessary, due to the (apparent) deprecation of static interfaces) without being sure that what I'm doing is correct.

Having said all that, I'm happy for Casset to be listed on packagist as as fuelphp 1.x package.

**Comment by dmyers** [comment]
FuelPHP 1.6 released recently requires composer. I think this would be a nice addition, but the PSR-0 is a good point.

http://fuelphp.com/blogs/2013/05/fuelphp-releases-v1-6
